validation/validation.go

```diff
@@ -27,9 +27,9 @@ func Validate(cfg *config.Local) []error {
 		errs = append(errs, err)
 	}
 
-	if err := validateIssueTrackerExists(cfg); err != nil {
```

Comment:
Shouldn't this work correctly when the token is provided?
And if not, I think removing the code completely without commenting it out could be a better approach. I think we should leverage git features for this sort of works.

Reply:
> Shouldn't this work correctly when the token is provided?
Yes, it should. The problem is that it currently doesn't take the token into consideration when making the HTTP request to check for the existence of the repository.

That's what I aim to address in a follow-up PR.

> And if not, I think removing the code completely without commenting it out could be a better approach. I think we should leverage git features for this sort of works.

What do you mean? How could we leverage git features to check for the existence of the repository?

Reply:
> Yes, it should. The problem is that it currently doesn't take the token into consideration when making the HTTP request to check for the existence of the repository.
> 
> That's what I aim to address in a follow-up PR.
Got it. It's alright then.

> What do you mean? How could we leverage git features to check for the existence of the repository?

No No. I meant **commenting out** the code is a bad practice and instead we should rely on git history for codes we do not need right now but maybe someday in the future.
So I suggest removing these lines and adding them again when we actually need it.

Reply:
Alright, makes sense